tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/event.go

```diff
@@ -0,0 +1,261 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	apiv1 "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+
+	subApis "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+	ebClientSet "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset"
+
+	"github.com/avast/retry-go"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+	eventingclientv1alpha1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+const timeout = time.Second * 30
+
+func SendEvent(target, eventType, eventTypeVersion string) error {
+	log.Printf("Sending an event to target: %s with eventType: %s, eventTypeVersion: %s", target, eventType, eventTypeVersion)
+	payload := fmt.Sprintf(
+		`{"event-type":"%s","event-type-version": "%s","event-time":"2018-11-02T22:08:41+00:00","data":"foo"}`, eventType, eventTypeVersion)
+	transport := &http.Transport{
+		DialContext: (&net.Dialer{
+			Timeout: timeout,
+		}).DialContext,
+	}
+	client := http.Client{Transport: transport}
+	res, err := client.Post(target,
+		"application/json",
+		strings.NewReader(payload))
+	if err != nil {
+		return errors.Wrap(err, "HTTP POST request failed in SendEvent() ")
+	}
+
+	if err := verifyStatusCode(res, 200); err != nil {
+		return errors.Wrap(err, "HTTP POST request returned non-2xx failed in SendEvent() ")
+	}
+
+	return nil
+}
+
+func CheckEvent(target, eventType, eventTypeVersion string, retryOptions ...retry.Option) error {
```

Comment:
It's the same as above, just with a retry, right? If yes, let's consolidate the two (e.g. just use a `retry` boolean param)

Reply:
nope, its not:
send event: sends event to the mesh (200 means successfully sent)
check event: check if the subscriber got the event (no error: means: subscriber got the event that we sent)

missing documentation it is :D